keps/sig-storage/3333-reconcile-default-storage-class/README.md

```diff
@@ -352,68 +365,53 @@ enabled for the first time.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Unit tests will cover feature enablement/disablement.
+Unit tests cover feature enablement/disablement.
```

Comment:
Those aren't really testing feature enablement/disablement.
Those are testing feature itself (when the feature gate is on).

Feature enablement/disablement is the operation of enabling/disabling the feature (so changing the value of feature gate).

As mentioned in the comment in the KEP template, here is an example of such test:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282

Reply:
Thinking more about it - I agree with @jsafrane - there is no new field here and there is no "migration phase" per-se.

So yes - I don't think we need additional code changes - just please explain it better in here in the text (roughly what @jsafrane wrote above).

Reply:
Ok, I've added an explanation and closed the PR linked above.